services/web/server/tests/integration/02/test_rabbit.py

```diff
@@ -344,16 +419,14 @@ async def test_publish_to_user(
     not_current_project_id: ProjectID,
     not_in_project_node_uuid: NodeID,
     #
-    socketio_subscriber_handlers: NamedTuple,
+    socketio_subscriber_handlers: SocketIoHandlers,
     publish_some_messages_in_rabbit: Callable[
         [UserID, ProjectID, NodeID, int],
-        Awaitable[tuple[LogMessages, ProgressMessages, InstrumMessages]],
+        Awaitable[tuple[LogMessages, ProgressMessages, InstrumMessages, EventMessages]],
     ],
 ):
-    mock_log_handler, mock_node_update_handler = socketio_subscriber_handlers
-
     # publish messages with correct user id, but no project
-    log_messages, _, _ = await publish_some_messages_in_rabbit(
+    log_messages, *_ = await publish_some_messages_in_rabbit(

```

Comment:
this is to do with tuple unpacking. here I am only interested in the first element of the tuple, not the others. So you can do either something like
```python
log_messages, _, _, _ = await publish_some_messages_in_rabbit() # there are 4 elements in the tuple
log_messages, *_ = await publish_some_messages_in_rabbit() # is the same as above but a bit more compact, and also adapts in case we add more elements.
```